src/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+:root {
+  --color-card-code: #616070;
+  --color-card-review: #060b35;
+  --color-buy-bacground: #00acdc;
+  --color-buy-text: white;
```

Comment:
Good practice is to name colors with color name like: **--color-white, --color-blue**

Reply:
bump consider using more descriptive names for colors, look at Dorota's mention before.

It's good practice because when someone went to your code and sees `--color-buy-text` will be confused about what it point to, and it is just plain white color :) 

e.g just use in that case `--color-white`